db/migrate/20181210151520_add_index_to_subscriptions.rb

```diff
@@ -0,0 +1,7 @@
+class AddIndexToSubscriptions < ActiveRecord::Migration[5.2]
+  def change
+    add_index :subscriptions, [:email, :status, :frequency, :expires_on], unique: true,
```

Comment:
this needs to be removed as it will restict creating multiple alerts on the same day (as search_criteria is a json field and cannot be used as part of the index)